qiskit/ignis/verification/accreditation/qotp.py

```diff
@@ -0,0 +1,250 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+# pylint: disable= no-member
+"""
+Applies a Quantum One Time Pad (QOTP)
+
+This is essentially the same protocol as used in
+randomized compiling, but follows the methods in
+
+https://iopscience.iop.org/article/10.1088/1367-2630/ab4fd6
+"""
+import numpy as np
+from qiskit import QuantumCircuit
+from qiskit.converters.circuit_to_dag import circuit_to_dag
+from qiskit.converters.dag_to_circuit import dag_to_circuit
+from qiskit.compiler import transpile
+
+
+def layer_parser(circ, twoqubitgate='cx', coupling_map=None):
```

Comment:
```suggestion
def layer_parser(circ, two_qubit_gate='cx', coupling_map=None):
```

Reply:
I just noticed in other parts of the code that call this function the default value is 'cz' but here the default value is 'cx'.

Reply:
In the code that sam and animesh wrote the assume cz is the standard gate.  It is not for us.  I guess I'll change it to cx everywhere.

Reply:
Either would be fine, but should be consistent throughout. Probably 'cx' makes most sense in Qiskit though.